src/static/js/routes/the-bureau/bureau-structure/index.js

```diff
@@ -8,22 +8,29 @@
 var $ = require( 'jquery' );
 require( 'slick' );
 
-var ContentSlider = require( '../../../modules/ContentSlider' );
-var BreakpointHandler = require( '../../../modules/BreakpointHandler' );
+var ContentSlider = require( '../../../modules/classes/ContentSlider' );
+var BreakpointHandler = require( '../../../modules/classes/BreakpointHandler' );
 
 function init() {
-  if ( !$( 'html' ).hasClass( 'lt-ie9' ) ) {
-    var settings = {
-      breakpoint: 599,
-      type:       'max',
-      enter:      _createSlider,
-      leave:      _destroySlider
-    };
-
-    var handler = new BreakpointHandler( settings );
+  if ( $( 'html' ).hasClass( 'lt-ie9' ) ) {
+    return;
   }
 
+  // TODO: Remove this when per-page JS is introduced.
+  if ( document.querySelectorAll( '#content-slider' ).length === 0 ) {
+    return;
+  }
+
+  var bpBettings = {
```

Comment:
No all good. I was confused. Yeah they share the same code. Updated in https://github.com/cfpb/cfgov-refresh/commit/5ee8753e588140c3eeb0298bdc4983182f806b9f